### PR TITLE
🌱 Update CI triggers after migrating to prow jenkins operator

### PR DESCRIPTION
**Pull request by mboukhalfa**
Update e2e test triggers after migrating to prow.
To check prow test list: `/test ?`

### CODE REVIEW

**Review by lentzi90 — COMMENTED** [review]
/approve

File: README.md [hunk 1]
```diff
@@ -51,31 +51,37 @@ For more details on the e2e tests, see [test/e2e/README.md](test/e2e/README.md).
...
+  integration tests with CAPM3 release-1.5 branch and BMO **release-0.4**
+  branch on Ubuntu
 
 **Release-0.3 branch:**
 
-- **/test-centos-e2e-integration-release-1-4** runs CAPM3 e2e integration tests
-  with CAPM3 release-1.4 branch and BMO **release-0.3** branch on Centos
-- **/test-ubuntu-integration-release-1-4** runs ansible integration tests with
-  CAPM3 release-1.4 branch and BMO **release-0.3** branch on Ubuntu
+- **/test metal3-centos-e2e-integration-test-release-1-4** runs CAPM3 e2e
```
Comment (tuminoid) [inline README.md:79]: Replace this with 0.6 branch, 0.3 is no longer supported or CI tested.
/hold

Comment (mboukhalfa) [reply README.md:79]: Do we have release-1.7 test running now ?

Comment (mboukhalfa) [reply README.md]: done

**Comment by mboukhalfa** [comment]
/hold cancel 
@tuminoid can you re lgtm

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Comment by tuminoid** [comment]
/retitle 🌱 Update CI triggers after migrating to prow jenkins operator

**Pull request merged**